spec/javascripts/helpers/SpecHelper.js

```diff
@@ -36,3 +36,19 @@ afterEach(function () {
 function isElementVisible(element) {
   return !!( element.offsetWidth || element.offsetHeight || element.getClientRects().length)
 }
+
+function isElementDisabled(selector) {
+  return document.querySelector(selector+':disabled') !== null
+}
+
+function isElementHidden(element) {
+  return !this.isElementVisible(element)
+}
+
+function isClassOnElement(element, className) {
+    return element.classList.contains(className)
+}
+
+function setFixtures(html) {
+  return jasmine.getFixtures().proxyCallTo_('set', arguments)
+}
```

Comment:
Just had a thought: could we put these functions into a `specHelpers` object? I think it might make the file look a little cleaner/help with readability.

Reply:
Thanks @JamesCGDS , i've done this now